Sources/FioriARKit/Utils/Environment+Extensions.swift

```diff
@@ -53,5 +58,9 @@ struct ActionTextModifierKey: EnvironmentKey {
 }
 
 struct CarouselOptionsKey: EnvironmentKey {
-    static let defaultValue = CarouselOptions(itemSpacing: 36, alignment: .bottom)
+    public static let defaultValue = CarouselOptions(itemSpacing: 36, alignment: .bottom)
```

Comment:
Clarification Question: the previous internal access level of `defaultValue` can be considered a bug? How was the app developer impacted?

Reply:
I changed it to be consistent with the other `EnvironmentKeys`. It might not technically matter if they are internal or public other than for formatting preference. The default will be used either way unless they use the carouselOptions View modifier to change the environment value as long as the extension function for view and `CarouselOptions` are public. The `EnvironmentKey` structs themselves can even be private since they are accessed through key paths. So this can be changed if you prefer.

What does matter is the access level of the `EnviromentValues` which determines if they are able to use what the sdk has defined with use of the Environment Property Wrapper.